crime-applications-adaptor/src/main/java/uk/gov/justice/laa/crime/applications/adaptor/service/EformStagingService.java

```diff
@@ -14,23 +15,29 @@
 @RequiredArgsConstructor
 @Slf4j
 public class EformStagingService {
-    private static final String EXCEPTION_MESSAGE_FORMAT = "USN: %d already used on MAAT Id: %d";
+    private static final String EFORM_STAGING_HUB_USER = "HUB";
+    private static final String EXCEPTION_MESSAGE_FORMAT = "USN: %d created by HUB user";
     private static final String SERVICE_NAME = "eformStagingService";
 
     private final MaatCourtDataApiClient eformStagingApiClient;
 
     private final ObservationRegistry observationRegistry;
 
-    @Retry(name=SERVICE_NAME)
-    public void retriveOrInsertDummyUsnRecord(Long usn) {
+    @Retry(name = SERVICE_NAME)
+    public EformStagingResponse retriveOrInsertDummyUsnRecord(Long usn) {
         log.info("Start - call to Eform Staging API ");
         EformStagingResponse eformStagingResponse = eformStagingApiClient.retrieveOrInsertDummyUsnRecordInEformStaging(usn);
-        Integer maatRef = eformStagingResponse.getMaatRef();
-        if (maatRef != null) {
-            String message = String.format(EXCEPTION_MESSAGE_FORMAT, maatRef, usn);
+        if (isUsnInEformStagingCreatedByHubUser(eformStagingResponse)) {
+            String message = String.format(EXCEPTION_MESSAGE_FORMAT, usn);
             throw new CrimeApplicationException(message);
         }
-        Observation.createNotStarted(SERVICE_NAME, observationRegistry)
-                .observe(()->eformStagingResponse);
+        return Observation.createNotStarted(SERVICE_NAME, observationRegistry)
+                .observe(() -> eformStagingResponse);
+    }
+
+    private static boolean isUsnInEformStagingCreatedByHubUser(EformStagingResponse eformStagingResponse) {
+        return StringUtils.isNotBlank(eformStagingResponse.getUserCreated())
```

Comment:
Can replace this with StringUtils like below

`org.apache.commons.lang3.StringUtils.equalsIgnoreCase(eformStagingResponse.getUserCreated(), EFORM_STAGING_HUB_USER);`

> Compares two CharSequences, returning true if they represent equal sequences of characters, ignoring case.
> nulls are handled without exceptions. Two null references are considered equal. The comparison is case insensitive.
>   StringUtils.equalsIgnoreCase(null, null)   = true
>   StringUtils.equalsIgnoreCase(null, "abc")  = false
>   StringUtils.equalsIgnoreCase("abc", null)  = false
>   StringUtils.equalsIgnoreCase("abc", "abc") = true
>   StringUtils.equalsIgnoreCase("abc", "ABC") = true
>   
> Params:
> cs1 – the first CharSequence, may be null cs2 – the second CharSequence, may be null
> Returns:
> true if the CharSequences are equal (case-insensitive), or both null
> Since:
> 3.0 Changed signature from equalsIgnoreCase(String, String) to equalsIgnoreCase(CharSequence, CharSequence)

Reply:
Agree.

Reply:
Change looks good